src/plugins/manualRowMove/test/manualRowMove.e2e.js

```diff
@@ -1387,4 +1387,58 @@ describe('manualRowMove', () => {
       });
     });
   });
+
+  describe('integration', () => {
+    it('should properly render saved order if columnSorting and persistentState are enabled', async() => {
+      const config = {
+        data: Handsontable.helper.createSpreadsheetData(5, 1),
+        persistentState: true,
+        manualRowMove: true,
+        columnSorting: true,
+      };
+      const container = spec().$container[0];
+      let hot = new Handsontable(container, config);
+
+      const plugin = hot.getPlugin('manualRowMove');
+
+      plugin.moveRow(0, 4);
+      plugin.persistentStateSave();
+
+      hot.destroy();
+
+      hot = new Handsontable(container, config);
+
+      await sleep(500);
```

Comment:
Is this sleep really needed? I'm not sure about it.

Reply:
I suggest leaving this `sleep`. Up to v7 columnSorting has been using ObserveChanges - without `sleep` was impossible to catch that problem.
I know ObserveChanges is now deprecated, but remember that scenario should point that we improved its behaviour between v7.4.2 and v8.0.0.

Reply:
So maybe this sleep could be much faster? It's quite a small dataset (`5x1`).

Reply:
The dataset's size doesn't matter, it's about unpredictable observe changes on CI. I tried 200ms and it worked locally but for CI env was too quick.